BetterReads/BetterReads/Views/SearchResultView.swift

```diff
@@ -33,27 +51,56 @@ class SearchResultView: UIView {
         setUpSubviews()
     }
     
+    private func updateViews() {
+        guard let book = book else { return }
+        
+        titleLabel.text = book.title
+        authorLabel.text = book.author
+        imageView.image = UIImage(systemName: book.cover)
+        //ratingView.text = "\(book.rating)"
+    }
+    
+    private func updateStarRating() {
+        print("updateStarRating")
+        for star in starsArray {
+            if star.tag <= Int(value) {
+                star.image = UIImage(systemName: "star.fill")
+                star.tintColor = .red
+            } else {
+                star.image = UIImage(systemName: "star")
+                star.tintColor = .green
+            }
+        }
+    }
+    
     private func setUpSubviews() {
+        // DELETE
+        value = 3.0
```

Comment:
Why does this say Delete?
Does this value need to be deleted?

Reply:
temporary value, changed